dpgen/dispatcher/Dispatcher.py

```diff
@@ -233,7 +254,7 @@ def all_finished(self,
                     job_record.record_finish(cur_hash)
                     job_record.dump()
         job_record.dump()
-        return job_record.check_all_finished()
+        return job_record.check_all_finished(), len(job_record.record)
```

Comment:
I think modifying return values in `all_finished` is a breaking change for check_dispatcher_status in DispatcherList